<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGithub"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfig"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGithub">
   <property name="icon">
    <iconset>
     <normaloff>Assets/icons/github.svg</normaloff>Assets/icons/github.svg</iconset>
   </property>
   <property name="text">
    <string>Github</string>
   </property>
   <property name="shortcut">
    <string>Alt+G</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>Assets/icons/door-open-line.svg</normaloff>Assets/icons/door-open-line.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset>
     <normaloff>Assets/icons/settings-3-line.svg</normaloff>Assets/icons/settings-3-line.svg</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
